### PR TITLE
refactor(response): making readable

### PR DESCRIPTION
**Pull request by Moore2021**

### CODE REVIEW

**Comment by Moore2021** [comment]
Purpose of this is to refactor the response lib, to try and stream line the function calls

File: src/libs/response.js [hunk 1]
```diff
@@ -71,39 +49,46 @@ class Response {
...
+	 * @param {String} plugins.file.filePath local file file path
+	 * @param {String} plugins.file.fileName local file name
+	 * @param {String} plugins.file.fileDescription local file description
 	 * @return {void}
 	 */
 	async send(content = ``, plugins = {}) {
-	
 		let socket = plugins.socket || []
 		let color = plugins.color || palette.crimson
-		plugins.color = color
```
Comment (klerikdust) [inline src/libs/response.js:106]: @Moore2021 pan are you really sure removing this line? i believe ~~some years ago~~ this was for patch fix the default embed color / profile card's main accent

**Review by klerikdust — APPROVED** [review]
holy skimming through hundred lines
but good dev QoL updates lol 🎉

File: src/libs/response.js (hunk 1 above)
Comment (Moore2021) [reply src/libs/response.js:106]: I don't know why it was there prior; initial testing hasn't found any issues but if you can remember where the error occurred and we can reproduce I'll gladly reconfigure that.

**Pull request merged**